<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rounds :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Player &amp;1 Name : </string>
     </property>
     <property name="buddy">
      <cstring>player1Name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Player &amp;2 Name :</string>
     </property>
     <property name="buddy">
      <cstring>player2Name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="player1Name"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="player2Name"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="numberRounds">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the number of rounds to win a game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>player1Name</tabstop>
  <tabstop>player2Name</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player1Name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOkButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player2Name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>updateOkButtonState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOkButtonState()</slot>
 </slots>
</ui>
